Jenkinsfile

```
@@ -1103,5 +1144,18 @@ pipeline {
                 }
             }
         }
+        stage("Performance Test - HIP") {
+            when {
+                expression {params.PERF_TEST && params.TARGET_GFX90A}
+            }
+            parallel{
+                stage('Performance Test Resnet50'){
+                    agent{ label rocmnode("austin")}
```

```suggestion
                    agent{ label rocmnode("gfx90a")}
```
Or the test can be run on some dedicated node(s) only?

---

we have issues with some of the 90a nodes not letting http traffic through, so we restricted it to Austin local 90a nodes that do allow it.

---

Proposal:
- In Jenkins, set a new label, `PERFORMANCE`, on nodes that allowed to be used in performance tests
- Use construct like `agent{ label rocmnode("PERFORMANCE && gfx90a") }`

---

You can temporarily use `agent{ label rocmnode("austin && gfx90a") }` of course.

---

I consider this as resolved.